meeting_minutes/2024/2024-02-28.md

```diff
@@ -68,23 +68,23 @@ Note: Observers of this committee that are ready to become Members should follow
   - Omar to merge after the call. 
 
  
-- [Pull Request 693 ](https://github.com/oasis-tcs/csaf/pull/707) and [Pull Request 694](https://github.com/oasis-tcs/csaf/pull/694) in version 2.1.
+- [Issue 693 ](https://github.com/oasis-tcs/csaf/issues/693) and [Issue 694](https://github.com/oasis-tcs/csaf/issues/694) in version 2.1.
   - TC should fix in 2.1 or another version.
-  - For current implementations, a router may be needed.
+  - For current implementations, a router <!-- Just marking here that this makes no sense to me. Maybe it should refer to an errata? --> may be needed.
```

Comment:
This is a point for discussion - I think errata might be the correct wording instead of "router"

Reply:
Close captioning for precedence 😁